src/controllers/controller_impls/project_controller.rs

```diff
@@ -82,7 +88,15 @@ impl ProjectControllerTrait for ProjectController {
                         let session = self
                             .contexts
                             .session_context
-                            .get_by_token(TokenType::AccessToken, request.token_string().unwrap())
+                            .get_by_token(
+                                TokenType::AccessToken,
+                                request
+                                    .token_string()
+                                    .map_err(|_err| {
+                                        Status::invalid_argument("token string from request metadata could not be stringified")
+                                    })?
+                                    .ok_or(Status::invalid_argument("token string from request metadata could not be stringified"))?,
+                            )
```

Comment:
Denne error er hvis `token_string` værdien er `None`

```suggestion
                                    .ok_or(Status::invalid_argument("failed to get token from request metadata"))?,
                            )
```